cookbook/bundles/installation.rst

```diff
@@ -68,22 +71,46 @@ The only thing you need to do now is register the bundle in ``AppKernel``::
         }
     }
 
+By default, Symfony bundles are registered in all the application
+:doc:`execution environments </cookbook/configuration/environments>`. If the bundle
+is meant to be used only in the development or test environments, register it in
+the section below::
```

Comment:
I would replace this sentence by "If the bundle is meant to be used only in some environment, register it within an if statement, like this (where the FOSUserBundle is only available in the dev and test environment)::"